lnbits/extensions/boltz/templates/boltz/_api_docs.html

```diff
@@ -78,7 +79,9 @@ <h3 class="text-subtitle1 q-my-none">
         <li>Onchain lockup tx fee: whatever you choose when paying</li>
         <li>Onchain claim tx fee: ~500 sats</li>
         <li>Routing fees (paid by boltz): unknown</li>
-        <li>Boltz fees: 500 sats (0.5%)</li>
+        {% raw %}
+        <li>Boltz fees: 500 sats ({{ boltzConfig.fee_percentage }}%)</li>
```

Comment:
Why is 500 fixed but the fee percentage can change?

Reply:
I agree completely. This should be calculated dynamically too!

Reply:
fixed it :)